whipper/test/test_common_mbngs.py

```diff
@@ -26,6 +26,24 @@ def testMissingReleaseDate(self):
 
         self.assertFalse(metadata.release)
 
+    def testTrackTitle(self):
+        """
+        Check if the "track title" is not "recording title"
```

Comment:
Just a nitpick but I'll reword the docstring like this:

>Check that the track title metadata is taken from MusicBrainz's track
>title (which may differ from the recording title, as in this case).

Reply:
Previous comment updated.

Reply:
Hi @JoeLametta , i have updated it like you suggested